repository/repository/4.root.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
Small thing, I thought repository/repository was changed in a previous PR to just repository?

Reply:
to repostiroy/repository: https://github.com/sigstore/root-signing/pull/303/files

this is how the sync job assumes the structure is too! 
https://github.com/sigstore/root-signing/blob/86a4cff5f890f5135e7d4f904e8091d948dfcfcf/.github/workflows/sync.yml#L56

Reply:
it also makes it easy for me to pull key info for the current published repository in a different subfolder.

renaming the top-level repository might be worth it... e.g. `current/`